python/Ganga/Lib/Batch/Batch.py

```diff
@@ -164,8 +164,7 @@ def submit(self, jobconfig, master_input_sandbox):
         if jobnameopt and job.name != '':
             # PBS doesn't like names with spaces
             tmp_name = job.name
-            from Ganga.GPI import PBS
-            if isType(self, PBS):
+            if self._name == "PBS":
```

Comment:
The old check probably never worked as `self` would never be a proxy type. Either way, this should be changed to just:
```python
 if isType(self, PBS):
```
since `PBS` is a valid type in this scope.